modules/installation/proc-config-rhdh-custom-resource.adoc

```diff
@@ -1,4 +1,53 @@
 [id="proc-config-rhdh-custom-resource_{context}"]
 = Configuring the {product-short} Custom Resource
 
-Updates to the Backstage custom resource (CR) are automatically handled by the {product} Operator. However, updates to resources referenced by the CR, such as ConfigMaps or Secrets, are not updated automatically unless the CR itself is updated. If you want to update resources referenced by the CR, then you must manually delete the Backstage Deployment so that the Operator can re-create it with the updated resources.
\ No newline at end of file
+You can change the configuration of the Custom Resource (CR) to program the way that the Operator manages your {product} instance.
+
+[NOTE]
+====
+Updates to the Backstage CR are automatically handled by the {product} Operator. However, updates to resources referenced by the CR, such as ConfigMaps or Secrets, are not updated automatically unless the CR itself is updated. If you want to update resources referenced by the CR, then you must manually delete the Backstage Deployment so that the Operator can re-create it with the updated resources.
+====
+
+.Prerequisites
+* You have an administrative role in {ocp-brand-name}
+* You have added a custom application configuration file to your {ocp-short} instance
+
+.Procedure
+
+To configure your {product-short} CR file for your {product-short} instance, complete the following steps:
+
+. Add a custom application configuration file to {ocp-short} and reference it in the CR. In {ocp-short}, you can use the following example as a base template to create a ConfigMap, such as `app-config-rhdh.yaml`:
```

Comment:
Where is explained how to reference the custom configuration file in the CR?

EDIT: I can see it in "Adding a custom application configuration file to OpenShift Container Platform using the Operator" (Step 3), but the text (`Select the YAML view option in Configure via and make the changes to the file, if necessary.`) should instruct to copy and adjust based on the content of the ConfigMap template below, IMO.